README.md

```diff
@@ -1,10 +1,30 @@
 # Public Allocator
 
-```bash
-$ forge test
-```
-Running the tests this way will use this repository's compilation settings for dependencies (Morpho & MetaMorpho), potentially different from the settings used for deployment of those dependencies.
+## Overview
+
+[MetaMorpho](https://github.com/morpho-org/metamorpho) is a protocol for funds allocations and risk management on top of [Morpho Blue](https://github.com/morpho-org/morpho-blue).
+The Public Allocator is a contract that MetaMorpho vault owners can enable as [allocator](https://github.com/morpho-org/metamorpho?tab=readme-ov-file#allocator) to let anybody reallocate the vault's funds to fill their liquidity needs.
+
+The Public Allocator's function `reallocateTo` ([source](https://github.com/morpho-org/public-allocator/blob/7271fbd60881ff32a466a588f99344c6bf72629a/src/PublicAllocator.sol#L108), [interface](https://github.com/morpho-org/public-allocator/blob/7271fbd60881ff32a466a588f99344c6bf72629a/src/interfaces/IPublicAllocator.sol#L62)) reallocates some liquidity from multiple markets to one market.
+It takes as input `withdrawals`, a list of pairs, which will be the markets and amounts that will be reallocated, and `supplyMarketParams`, a `MarketParams`, which is the market in which to supply the reallocated funds.
+Note that the `id` of the markets in `withdrawals` must be sorted, and that `supplyMarketParams` cannot be a market of `withdrawals`.
```

Comment:
```suggestion
Note that the `id` of the markets in `withdrawals` must be sorted in ascending alphabetical order, and that `supplyMarketParams` cannot be a market of `withdrawals`.
```

Reply:
I think it's confusing to say `alphabetical` here, because it's really the usual order on hex number. For example `0xa0` is greater than `0xb` (meaning `0x0b`) but you would find it first in a dictionary

Reply:
Right, just to make sure our offchain code is not completely off: for same-length hex strings, the expected order does correspond to the alphabetical order right?

Reply:
Yes, because the alphabetical order is the same as the (usual) numerical order for numbers of the same length

Reply:
But what's before between 0x9 and 0xa in "alphabetical" order ? Better not to put it I think